druid-shell/Cargo.toml

```diff
@@ -57,3 +59,9 @@ gtk-sys = "0.9.0"
 [target.'cfg(target_os="linux")'.dependencies.gtk]
 version = "0.8.1"
 features = ["v3_20"]
+
+[target.'cfg(target_arch="wasm32")'.dependencies]
+wasm-bindgen = "0.2.59"
+js-sys = "0.3.36"
+web-sys = { version = "0.3.36", features = ["Window", "MouseEvent", "CssStyleDeclaration", "WheelEvent", "KeyEvent", "KeyboardEvent"] }
+console_log = { version = "0.1.2", optional = true }
```

Comment:
I'm okay with this being required, if we can make it work transparently with `log` and with our `use_simple_logger` method?